### PR TITLE
Example 1 edit/proofing

### PR DESCRIPTION
**Pull request by humanfactors**
Apologies for the lack of isolation - it's much harder to commit in chunks when getting in the flow of editing writing. I'm going to work out a better workflow with this over time. Please also let me know if this is the appropriate workflow to get this onto the extant PR stack into master?

**Overall the commit is largely just stylistic and grammatical changes:** ✍✔

* I've adjusted a number of sentences across the example to read more clearly. 👁‍🗨
* Any explicit reference to variables now is quoted in code block tags rather than bolding to make more clear it's part of the R code.
* I've change the plot colors to be a little more readable on the eyes (and improved for color blindness)
* Added in blank lines between code blocks for source readability
* Capitalization of Sunflower 🌻🌻🌻

**General Points**
- I'm not a huge fan of the use of bolding throughout, as I think the emphasis is actually lost with excessive emphasis. I've sparingly removed some of these.
- I've tried to ensure that when a variable name is referred to in text, it's in verbatim code tags.

Sorry, I'm confused by the presence of 13e920c27827bb60656d50d15b3031c07c37d942 in the PR as that was my fork updating to latest dev.

### CODE REVIEW

File: vignettes/example1.Rmd [hunk 1]
```diff
@@ -91,20 +90,21 @@ You can see the sampling algorithm being run.
...
 ```
+
 ```{r message=FALSE, warning=FALSE, echo=FALSE}
 posteriors <- insight::get_parameters(model)
 
 head(posteriors)  # Show the first 6 rows
 ```
 
-As we can see, the parameters take the form of a long dataframe with two columns, corresponding to the **intercept** and the effect of **Petal.Length**. These columns contain the **posterior distributions** of these two parameters, *i.e.*, a set of different plausible values.
+As we can see, the parameters take the form of a wide dataframe with two columns, corresponding to the intercept and the effect of `Petal.Length`. These columns contain the **posterior distributions** of these two parameters. In simple terms, the posterior distribution is a set of different plausible values for each parameter.
```
Comment (DominiqueMakowski) [inline vignettes/example1.Rmd]: actually here the dataframe is "long" (i.e. more rows than columns) ☺️ 

I would also add batsticks around `intercept` as it refers to the parameter name

File: vignettes/example1.Rmd [hunk 2]
```diff
@@ -116,54 +116,57 @@ nrow(posteriors)  # Size (number of rows)
...
 
-
-Now that we've understood where these values come from, let's look at them. We will start by visualizing the distribution of our parameter of interest, the effect of `Petal.Length`.
+Now that we've understood where these values come from, let's look at them. We will start by visualizing the posterior distribution of our parameter of interest, the effect of `Petal.Length`.
 
 
 ```{r message=FALSE, warning=FALSE}
 ggplot(posteriors, aes(x = Petal.Length)) +
-  geom_density(fill = "orange")
+  geom_density(fill = "wheat")
```
Comment (DominiqueMakowski) [inline vignettes/example1.Rmd]: 😄

File: vignettes/example1.Rmd [hunk 3]
```diff
@@ -116,54 +116,57 @@ nrow(posteriors)  # Size (number of rows)
...
 ```
 
 This distribution represents the [probability](https://en.wikipedia.org/wiki/Probability_density_function) (the y axis) of different effects (the x axis). The central values are more probable than the extreme values. As you can see, this distribution ranges from about **0.35 to 0.50**, with the bulk of it being at around **0.41**.
 
 > **Congrats! You've just described your posterior distribution.**
 
-And this is at the heart of Bayesian analysis. We don't need *p*-values, *t*-values or degrees of freedom: **everything is there**, within this **posterior distribution**.
+And this is at the heart of Bayesian analysis. We don't need *p* values, *t* values or degrees of freedom: **everything is there**, within this posterior distribution.
```
Comment (DominiqueMakowski) [inline vignettes/example1.Rmd]: throughout most of the package, we spell *p*-values (I think it is the APA-like spelling), so I suggest keeping the `-`

File: vignettes/example1.Rmd [hunk 4]
```diff
@@ -116,54 +116,57 @@ nrow(posteriors)  # Size (number of rows)
...
 We can now go ahead and **precisely characterize** this posterior distribution.
 
 ### Describing the Posterior
 
-Unfortunately, it is often not practical to report the whole posterior distributions as graphs. We need to find a **concise way to summarize it**. We recommend to describe the posterior distribution with **3 elements**. A **point-estimate**, a one-value summary (similar to the *beta* in frequentist regressions); a **credible interval**, representing the associated uncertainty and some **indices of significance**, giving information about the importance of this effect.
+Unfortunately, it is often not practical to report the whole posterior distributions as graphs. We need to find a **concise way to summarize it**. We recommend to describe the posterior distribution with **3 elements**:
+
+1. A **point-estimate** which is a one-value summary (similar to the *beta* in frequentist regressions).
+2. A **credible interval** representing the associated uncertainty.
+3. And some **indices of significance**, giving information about the relative importance of this effect.
```
Comment (DominiqueMakowski) [inline vignettes/example1.Rmd]: In this context, we can remove the "And" at the beginning of point 3

File: vignettes/example1.Rmd [hunk 5]
```diff
@@ -185,11 +188,11 @@ map_estimate(posteriors$Petal.Length)
 
 ```{r message=FALSE, warning=FALSE}
 ggplot(posteriors, aes(x = Petal.Length)) +
-  geom_density(fill = "orange") +
+  geom_density(fill = "wheat") +
   # The mean in blue
-  geom_vline(xintercept=mean(posteriors$Petal.Length), color="blue", size=1) +
```
Comment (DominiqueMakowski) [inline vignettes/example1.Rmd]: for these vline colors I think it's better to keep raw colours (blue and red) as it is how we refer to them after

File: vignettes/example1.Rmd [hunk 6]
```diff
@@ -257,25 +260,25 @@ model <- stan_glm(weight ~ feed, data=data)
...
 
 > **Is this effect different from 0?**
 
 ### ROPE Percentage
 
 Testing whether this distribution is different from 0 doesn't make sense, as 0 is a single value (*and the probability that any distribution is different from a single value is infinite*). 
 
-However, one way to assess **significance** could be to define an area around 0, which will consider as *equivalent* to zero (*i.e.*, absence of, or negligible, effect). This is called the [**Region of Practical Equivalence (ROPE)**](https://easystats.github.io/bayestestR/articles/region_of_practical_equivalence.html), and is one way of testing the significance of parameters.
+However, one way to assess **significance** could be to define an area around 0, which will consider as *practically equivalent* to zero (*i.e.*, absence of, or negligible, effect). This is called the [**Region of Practical Equivalence (ROPE)**](https://easystats.github.io/bayestestR/articles/region_of_practical_equivalence.html), and is one way of testing the significance of parameters.
```
Comment (DominiqueMakowski) [inline vignettes/example1.Rmd]: I am not sure here, as the practically might be redundant and confusing for non-english (in French, "practically" (pratiquement) refers both to "in a practical manner" and to "almost"). Hence, keeping here only equivalent avoids this problem (and the practically comes afterwards either way.

**Comment by humanfactors** [comment]
Sorry, I'm confused by the presence of 13e920c27827bb60656d50d15b3031c07c37d942 in the PR as that was my fork updating to latest dev.

**Comment by DominiqueMakowski** [comment]
> Sorry, I'm confused by the presence of 13e920c in the PR as that was my fork updating to latest dev.

It's normal, this way there are no conflicts between the branches

File: vignettes/example1.Rmd (hunk 1 above)
Comment (humanfactors) [reply vignettes/example1.Rmd]: My understanding is that long refers to the format of:

`MEASURE,VALUE` while wide basically means `ID, VAR1, VAR2` with values contained in columns?

I could be wrong, but I don't think it's about length of the total DF? That would be transposition wouldn't it?

File: vignettes/example1.Rmd (hunk 3 above)
Comment (humanfactors) [reply vignettes/example1.Rmd]: I'm happy to retain the `-` for consistency across the package, though I believe the APA does recommend no hyphen in 6th ed.

File: vignettes/example1.Rmd (hunk 5 above)
Comment (humanfactors) [reply vignettes/example1.Rmd]: Ahhh I missed this, good spotting.

File: vignettes/example1.Rmd (hunk 1 above)
Comment (DominiqueMakowski) [reply vignettes/example1.Rmd]: > MEASURE,VALUE while wide basically means ID, VAR1, VAR2 with values contained in columns?

When it refers to repeated measures yes that is correct

But in general a long dataframe can also refer to long in terms of length (number of observations), which is what we wanted to say here (there are 2 cols of 4000 obs)

File: vignettes/example1.Rmd (hunk 3 above)
Comment (DominiqueMakowski) [reply vignettes/example1.Rmd]: > I believe the APA does recommend no hyphen in 6th ed.

If that is confirmed we will need to slowly remove them in all easystats (as I meticulously placed them everywhere) ^^

File: vignettes/example1.Rmd (hunk 1 above)
Comment (strengejacke) [reply vignettes/example1.Rmd]: Maybe it's better to call it "large" or something, I found the term "long" also confusing (it somehow implies long versus wide format).

File: vignettes/example1.Rmd (hunk 6 above)
Comment (strengejacke) [reply vignettes/example1.Rmd]: We could keep practically here, as ROPE also has it in its name. I think here the redundancy serves a better understanding.

File: vignettes/example1.Rmd (hunk 1 above)
Comment (humanfactors) [reply vignettes/example1.Rmd]: I'll change to lengthy in next commit?

File: vignettes/example1.Rmd (hunk 3 above)
Comment (DominiqueMakowski) [reply vignettes/example1.Rmd]: (do you have a link where it says that?)

Comment (humanfactors) [reply vignettes/example1.Rmd]: image

I can't actually find an explicit reference in the guide, however, you can see here (APA Style Guide 6th Ed., p. 114) that the guide writes "_p_ value" without the hyphen. However, you do have to have the hyphen for "_t_-test". Nevertheless, I've removed in the commit as it's something we can do a find and replace for at a later date.

File: vignettes/example1.Rmd (hunk 6 above)
Comment (humanfactors) [reply vignettes/example1.Rmd]: I've kept in f047f6ff655ce87849a2a3ec23e33d88f14342aa

Comment (DominiqueMakowski) [reply vignettes/example1.Rmd]: alright

File: vignettes/example1.Rmd (hunk 3 above)
Comment (DominiqueMakowski) [reply vignettes/example1.Rmd]: thanks. Yeah, in that case for v3 we will remove the hyphen everywhere ☺️

**Comment by DominiqueMakowski** [comment]
If you're ok @humanfactors I can merge

**Comment by humanfactors** [comment]
All ok and happy to merge!

**Comment by humanfactors** [comment]
Accidental close there....

**Comment by DominiqueMakowski** [comment]
> Feel free to add yourself as ctb in the DESCRIPTION file

Now is the chance to push another commit :)

**Comment by humanfactors** [comment]
Squeezed in the cheeky commit there. Good call!

**Pull request merged**